src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js

```diff
@@ -163,7 +163,7 @@ define([
                     '/admin/api/media/' + curMediaId + '?action=new-version' :
                     '/admin/api/media?collection=' + this.formOptions.accountAvatarCollection;
                 
-                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.user.locale);
+                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.getDefaultContentLocale());
```

Comment:
But if the user locale is not also a content locale we are running into troubles here... When the first content language is one the user doesn't understand there is also no way for him to change that. Using the content locale before using the default locale is probably the better choice here.